06_PDDE_Checker.R

```diff
@@ -227,6 +227,28 @@ zero_utilization <- Project0 %>%
   mutate(Detail = "") %>%
   select(all_of(PDDEcols))
 
+
+# RRH-SO projects with active inventory -----------------------------------
+
+rrh_so_w_inventory <- Inventory %>%
+  mutate(
+    InventoryActivePeriod = 
+      interval(InventoryStartDate,
+               coalesce(InventoryEndDate, meta_HUDCSV_Export_End))
+  ) %>%
+  select(InventoryID, ProjectID, InventoryActivePeriod, BedInventory) %>%
+  left_join(Project, join_by(ProjectID)) %>%
+  mutate(RRHSOyn = ProjectType == 13 & RRHSubType == 1,
+         RRHSOActivePeriod =
+           interval(OperatingStartDate,
+                    coalesce(OperatingEndDate, meta_HUDCSV_Export_End)),
+         Detail = "") %>%
```

Comment:
Do we not want any detail here?

Reply:
I thought about it, looked at it in the output workbook and couldn't think of anything that would help beyond what's there but if you think of something lmk, we can always add it later! Thanks Alex!